app/models/theme.rb

```diff
@@ -288,21 +303,27 @@ def set_field(target:, name:, value: nil, type: nil, type_id: nil, upload_id: ni
 
   def all_theme_variables
     fields = {}
-    ([self] + (included_themes || [])).each do |theme|
-      theme&.theme_fields.each do |field|
-        next unless ThemeField.theme_var_type_ids.include?(field.type_id)
-        next if fields.key?(field.name)
-        fields[field.name] = field
-      end
+    ids = [self.id] + (included_themes&.map(&:id) || [])
+    _fields = ThemeField
```

Comment:
Hmm an underscore in front of a variable name usually means the variable is not being used. However, it seems like this is an actual variable which we want to use here.